web/static.go

```diff
@@ -139,3 +145,18 @@ func unsupportedBrowserScriptHandler(w http.ResponseWriter, r *http.Request) {
 	templatesDir, _ := templates.GetTemplateDirectory()
 	http.ServeFile(w, r, filepath.Join(templatesDir, "unsupported_browser.js"))
 }
+
+func getOGDescriptionMetaTag(c *Context) string {
+	siteDescription := model.TeamSettingsDefaultCustomDescriptionText
+	customSiteDescription := c.App.Srv().Config().TeamSettings.CustomDescriptionText
+	if customSiteDescription != nil && *customSiteDescription != siteDescription {
```

Comment:
I'm thinking this may be a better way to determine if we should use the custom description

```suggestion
        if customSiteDescription != nil && len(*customSiteDescription) > 0 {
```